hibike/setup.sh

```diff
@@ -41,7 +52,7 @@ if [ "$(dpkg-query -W -f='${Status}' python3-pip 2>/dev/null | grep -c "ok insta
     apt-get --assume-yes install python3-pip
 fi
 
-if [ "$(pip3 freeze | grep -c "pyserial")" != 1]; then
+if [ "$(pip3 freeze | grep -c "pyserial")" != 1 ]; then
```

Comment:
This is because we cherry picked some commits from the branch that pr came from. What's the best way to resolve these conflicts now?